rocketmq-doc/_docs/19-contribute-guide.md

```
@@ -0,0 +1,109 @@
+
+- Before submitting an issue, please go through a comprehensive search to make sure the problem cannot be solved just by
+  searching.
+- Check the [Issue List](https://github.com/mxsm/rocketmq-rust/issues) to make sure the problem is not repeated.
+- Create a new issue and choose the type of issue.
+- Define the issue with a clear and descriptive title.
+- Fill in necessary information according to the template.
+- Please pay attention for your issue, you may need provide more information during discussion.
+
+## How to Contribute
+
+#### 1. Prepare repository
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)